AutomatedTesting/Gem/PythonTests/assetpipeline/fbx_tests/assets/default_script/default_script.py

```diff
@@ -0,0 +1,56 @@
+#
+# Copyright (c) Contributors to the Open 3D Engine Project.
+# For complete copyright and license terms please see the LICENSE at the root of this distribution.
+#
+# SPDX-License-Identifier: Apache-2.0 OR MIT
+#
+#
+
+import os
+
+sceneJobHandler = None
+
+def clear_sceneJobHandler():
+    global sceneJobHandler
+    # do not delete or set sceneJobHandler to None, just disconnect from it.
+    # this call is occurring while the scene Job Handler itself is in the call stack,
+    # so deleting it here would cause a crash.
+    sceneJobHandler.disconnect()
+
+def on_prepare_for_export(args):
+    import azlmbr.scene
+    import azlmbr.object
+    import azlmbr.paths
+    import azlmbr.math
+
+    scene = args[0] # azlmbr.scene.Scene
```

Comment:
Nit: Is it worth checking the length of args before using list elements at some specific indexes here?

Reply:
honestly, if the args does not have a 0th element then the script will raise an exception which will fail the test